src/components/Catalog.vue

```diff
@@ -220,13 +251,26 @@ export default {
           sortable: true
         }
       ],
+      collectionFields: [
+        {
+          key: "link",
+          label: "Identifier",
+          sortable: false // Sorting doesn't work for links
+        },
+        {
+          key: "title",
+          label: "Title",
+          sortable: true
+        }
+      ],
       currentChildPage: 1,
-      childrenPerPage: 25,
+      currentCollectionPage: 1,
+      childrenPerPage: 25, // also applies to collections
       itemFields: [
         {
           key: "link",
           label: "Title",
-          sortable: true
+          sortable: false // Sorting doesn't work for links
```

Comment:
Sorting did not work at all yet for cells with links in them, so disabling for now until we have a better bug fix.